plugins/catalog-import/src/components/EntityListComponent/EntityListComponent.tsx

```diff
@@ -129,7 +138,11 @@ export const EntityListComponent = (props: EntityListComponentProps) => {
                 );
                 return (
                   <ListItem
-                    key={humanizeEntityRef(entity)}
+                    key={
+                      entityPresentationApi.forEntity(
+                        stringifyEntityRef(entity),
+                      ).snapshot.entityRef
+                    }
```

Comment:
```suggestion
                    key={stringifyEntityRef(entity)}
```

the keys don't matter, they just have to be unique (used only internally by React)